lib/net.js

```diff
@@ -1190,7 +1199,10 @@ function Server(options, connectionListener) {
       this.on('connection', connectionListener);
     }
   } else {
-    throw new TypeError('options must be an object');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE',
+                               'options',
+                               ['object'],
```

Comment:
s/`[object']`/`Object'

When there is a single value, it doesn't need to be in an array